### PR TITLE
i18n: Mark up HelpModal for translations

### LINKED ISSUE #73614: i18n: Mark up `HelpModal` for translations
help modal is accessed by pressing `h`:
image

follow instructions [here](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#how-to-add-a-new-translation-phrase). can verify translation strings have been marked up correctly by using the development-only pseudo locale: 
image

checklist:
- [ ] modal title
- [ ] close tooltip
- [ ] shortcut category titles
- [ ] shortcut descriptions

### PR DESCRIPTION
**Pull request by Dennnnny**
**What is this feature?**

mark up for translation in Help Modal and Modal close tooltip

**Why do we need this feature?**

To ensure all strings will be translated.

**Who is this feature for?**

Everyone.

**Which issue(s) does this PR fix?**:

Fixes #73614 

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: packages/grafana-ui/src/components/Modal/Modal.tsx [hunk 1]
```diff
@@ -84,7 +85,7 @@ export function Modal(props: PropsWithChildren<Props>) {
               typeof title !== 'string' && title
             }
             <div className={styles.modalHeaderClose}>
-              <IconButton name="times" size="xl" onClick={onDismiss} tooltip="Close" />
+              <IconButton name="times" size="xl" onClick={onDismiss} tooltip={t('modal.close-tooltip', 'Close')} />
```
Comment (joshhunt) [inline packages/grafana-ui/src/components/Modal/Modal.tsx:88]: this should be in our documentation, but can you please change this ID to `t('grafana-ui.modal.close-tooltip', 'Close')` and re-run `yarn i18n:extract`?

**Comment by Dennnnny** [comment]
sorry I'm not sure what and how to set a milestone 🤔 
any hints ?

File: packages/grafana-ui/src/components/Modal/Modal.tsx (hunk 1 above)
Comment (Dennnnny) [reply packages/grafana-ui/src/components/Modal/Modal.tsx]: done. thanks

**Comment by joshhunt** [comment]
@Dennnnny No need to worry about milestone - we'll take care of that

**Comment by Dennnnny** [comment]
not sure why this Check for patch conflicts test fail ...  🤔 
error message: `Error: Error: Input required and not supplied: app_id`

**Comment by tskarhed** [comment]
@Dennnnny It is a bit confusing, but you don't have to worry about the check for patch conflicts 😅 

The linting seems to fail. You can fix it by running prettier.

```
+ yarn run prettier:check
+ yarn run lint

/drone/src/packages/grafana-ui/src/components/Modal/Modal.tsx
  13:1  error  There should be at least one empty line between import groups                       import/order
  14:1  error  `../../utils/i18n` import should occur before import of `../IconButton/IconButton`  import/order

/drone/src/public/app/core/components/help/HelpModal.tsx
  7:1  error  `app/core/internationalization` import should occur before import of `app/core/utils/browser`  import/order

✖ 3 problems (3 errors, 0 warnings)
  3 errors and 0 warnings potentially fixable with the `--fix` option.
```

You will also need to sync this branch with the main branch.

**Comment by Dennnnny** [comment]
> @Dennnnny It is a bit confusing, but you don't have to worry about the check for patch conflicts 😅
> 
> The linting seems to fail. You can fix it by running prettier with the `--fix` option.
> 
> You will also need to sync this branch with the main branch.

thanks for help 😂  
I fix the lint and rebase to main. 
if there is anything need to adjust
please let me know.

thanks

**Comment by Dennnnny** [comment]
@tskarhed  hello, sorry I want to know one thing:
`continuous-integration/drone/pr` fail is because it needs to wait for review or it really fail test ?? 
( I'm new for contribution, so I got a little confused. TIA )

**Comment by tskarhed** [comment]
@Dennnnny It is for a test that seems unrelated. I'll try to rerun it, otherwise I'll ask around.

**Review by ashharrison90 — COMMENTED** [review]
@Dennnnny last thing to do is to run `yarn i18n:pseudo` so it generates some pseudo translations correctly. otherwise looks great 🙌 (aside: if you install our precommit hooks locally via `make lefthook-install` it will do this automatically for you on commit)

**Review by ashharrison90 — APPROVED** [review]
looks great! 🥳 

thanks so much for your contribution to Grafana @Dennnnny 🙇

**Pull request merged**